### PR TITLE
fix: update view post

### PR DESCRIPTION
**Pull request by ViNguyenHoang**
fix: update view post

### CODE REVIEW

File: api/main_service/MainService/Controllers/PostsController.cs [hunk 1]
```diff
@@ -162,15 +162,42 @@ public async Task<ActionResult<ResponseDto>> AddPost([FromForm] PostCreateDto ne
...
+        postFilter = postFilter & Builders<Post>.Filter.Lte(x => x.PublishedAt, DateTime.Now);
+
+        // Get User Id
+        try
+        {
+            userId = User.FindFirst(JwtTokenPayload.USER_ID)!.Value;
+        }
+#pragma warning disable CS0168
+        catch (Exception e)
+#pragma warning restore CS0168
```
Comment (marshmary) [inline api/main_service/MainService/Controllers/PostsController.cs:183]: What is this ?

**Review by marshmary — COMMENTED** [review]
Pls read my comment and reply to it

File: api/main_service/MainService/Controllers/PostsController.cs (hunk 1 above)
Comment (ViNguyenHoang) [reply api/main_service/MainService/Controllers/PostsController.cs:183]: A trick to disable warning of catch Exception variable which doesn't use

Comment (marshmary) [reply api/main_service/MainService/Controllers/PostsController.cs:183]: Did you remember that in C# try catch can be done without (Exception e) ?

**Pull request merged**